Documentation/freebsd-instructions.md

```diff
@@ -24,13 +24,33 @@ Install the following packages for the toolchain:
 - libunwind
 - gettext
 
-Note: LLDB is not yet available in the pkg repository on FreeBSD. The LLDB plugin is currently not being build on FreeBSD. Instructions for this will be added later.
-
 To install the packages you need:
 
-```janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake clang35 libunwind gettext```
+```$ sudo pkg install bash cmake clang35 libunwind gettext```
```

Comment:
For consistency I would keep the line from the original manual, jumping back and forth between different differently formatted example prompts can be confusing.